### ISSUE TITLE
VPAT: Dataset landing page

### ISSUE THREAD

**Issue by dlowenberg** [CDL-Dryad/dryad-product-roadmap#1328]
- [x] Adjust heading levels
- [x] Use h1 for the title and h2 for the heading in the Modal dialog (similar to the rest of the dialog suggestions)
- [x] Keep modal open until user closes it.

**Comment by sfisher** [comment]
I don't know what the spreadsheet is talking about with missing H3 and then H4.  Did they keep an old spreadsheet and not update or something?  Because there is stuff in there that is just wrong (such as this and also stuff about Geolocation which we haven't had for a long time.)

**Comment by sfisher** [comment]
I'm not going to touch the modal dialog until we discuss since they also have things about replacing all the modals in our project and seems like this is one of them that they'd like us to replace.  There is a bit of functionality included for downloading, so it's more involved than something that just gives a dialog and looks pretty.

We can certainly replace all modals in our project if we want to take the time and effort, but it will be it's own little project and I honestly have questions about the criticality of some of the things and also some of the rationales they give (like the idea that someone is would be confused about what a dialog is asking when the user triggered the action themselves).
